src/js/components/sharedComponents/filterSidebar/FilterExpandButton.jsx

```diff
@@ -5,19 +5,24 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { AngleRight, AngleDown, InfoCircle } from 'components/sharedComponents/icons/Icons';
+import { AngleRight, AngleDown } from 'components/sharedComponents/icons/Icons';
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const propTypes = {
     hideArrow: PropTypes.bool,
     toggleFilter: PropTypes.func,
     arrowState: PropTypes.string,
     name: PropTypes.string,
+    // functional component!
```

Comment:
is `PropTypes.element` bad or will not work for this?

Reply:
Yeah it throws an error in the conolse for some reason